data-pipeline/src/pipeline/database.py

```diff
@@ -192,6 +192,7 @@ def _write_data(
         con=engine,
         if_exists="append",
         index=df.index.name is not None,
+        chunksize=int(os.getenv("SQL_CHUNKSIZE", 10_000)),
```

Comment:
Is `SQL_CHUNKSIZE` defined somewhere already?

Reply:
It's reading from an environment variable; if it's set, it'll pick up that value, if not it'll default to 10,000.

I'd originally hard-coded 10,000 but added this in case we want to optimise in the future.

Reply:
Ah, OK. Wasn't sure if you wanted to support some custom value via env var from the outset